server/CONTRIBUTING.md

```diff
@@ -35,29 +36,30 @@ $ git clone git@github.com:[username]/rsschool-api.git
 $ cd rsschool-api
 $ npm install
 ```
-4. Create a branch for your feature
+4. If you plan to change the server part, please create a branch for your feature
 ``` command-line
 $ git checkout -b feature-x master
 ```
-5. The application requires a connection to a MongoDB database. There are 2 options:
+5. The application requires a connection to a Postgres database. There are 2 steps:
 
-    * You may specify the following environment variables in order to connect to database:
+    * You should specify the following environment variables in order to connect to database in command line or rename .env.example to .env (point 9 below):
     ``` command-line
-        RSSHCOOL_API_MONGO_CONNECTION_STRING
-        RSSHCOOL_API_MONGO_USER
-        RSSHCOOL_API_MONGO_PASSWORD
+        RSSHCOOL_API_PG_HOST
+        RSSHCOOL_API_PG_USERNAME
+        RSSHCOOL_API_PG_PASSWORD
+        RSSHCOOL_API_PG_DATABASE
     ```
 
-    * You may run MongoDB locally with pre-imported test data using [Docker Compose file](https://github.com/rolling-scopes/rsschool-scripts/blob/master/development/docker-compose.yml) from [rsschool-scripts](https://github.com/rolling-scopes/rsschool-scripts) repository.
-    Following commands will run MongoDB locally binded to 27107 port and will import test `Users` and `Courses` data.
+    * You should run Postgres locally with pre-imported test data using [Docker Compose file](https://github.com/rolling-scopes/rsschool-scripts/blob/master/development/docker-compose.yml) from [rsschool-scripts](https://github.com/rolling-scopes/rsschool-scripts) repository.
+    Following commands will run Postgres locally binded to 5432 port and will import test `Users` and `Courses` data.
```

Comment:
- `npm run db:up` runs database locally using docker
- `npm run db:restore` restore test copy of DB locally.

`rsschool-scripts` repo is not used anymore